src/containers/Gurb/SupplyPoint.jsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Header from './components/Header'
+import ProgressWarning from './components/ProgressWarning'
+import TextRecomendation from './components/TextRecomendation'
+import CUPS from './components/CUPS'
+import { getGurbData } from '../../services/apiGurb'
+
+const SupplyPoint = (props) => {
+  const { t } = useTranslation()
+  const [gurbData, setGurbData] = useState({})
+
+  const handleGurbData = async () => {
+    // TODO: gurb id from where?
+    const gurbId = 1
+    await getGurbData(gurbId)
+      .then(({ data }) => {
+        setGurbData({
+          name: data?.name,
+          state: data?.state,
+          completedPercentage: data?.assigned_betas_percentage
```

Comment:
We think that we should deal with the empty data and other errors at higher level, for instance, if there is no data a pop up will appear and show you the error (let's see what our UXer has to say on how to show the errors). Regarding the unexpected error: the idea of using `data?.name_of_field` is exactly that, it will return an undefined and will not through an error.